Dockerfile

```
@@ -2,13 +2,13 @@ FROM node:14-alpine AS build
 
 WORKDIR /opt/node_app
 
-COPY package.json package-lock.json ./
-RUN npm i --no-optional
+COPY package.json yarn.lock ./
+RUN yarn
```

```suggestion
RUN yarn --ignore-optional
```
to ignore options deps https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-ignore-optional